.golangci.yml

```diff
@@ -4,13 +4,15 @@ run:
 #   timeout: 5m
 
 linters:
-  disable-all: true
+  disable:
+    - errcheck
+    - wsl
+    - varcheck
```

Comment:
`errcheck` is the only needed here if you are not using `disable-all`. They practically do the same thing because the only linter from the defaults we keep disabled is `errcheck`. This is more of a preference thing.

Reply:
One of the motivation of this PR is to not use `disable-all`. From my OP:

> It's better to explicitly disable default lint checkers rather then disable all of them. The main reason is that if we golangci-lint will have a very good and useful checker we may skip him.

Reply:
Can we enable varcheck, also WSL is not needed here, it's not a default linter

Reply:
BTW: we are also not using `varcheck` - it's slow and `unused` is faster and check functions as well.

Reply:
I removed the line with wsl

Reply:
> It's better to explicitly disable default lint checkers rather then disable all of them.

I'd argue that no, that is not necessarily better, reason being:

> if we enable all checks by default, new checks may be introduced unpromptedly, causing PRs to fail unexpectedly